### DISCUSSION TITLE
eu compatibility ?

### DISCUSSION THREAD

**Discussion by AMiR-DU-Bled** [Admentus64/Patcher64Plus-Tool#165] [Ideas]
Hi can you please add eu support for the software because I want to play Europeans roms but the patcher wont work.

**Comment by Admentus64** [comment]
You can patch in PAL languages into the US ROM instead.

PAL support itself will not happen. Closing the discussion since there's nothing to discuss since PAL support is not going happen. PAL is too vastly inferior for N64 games at 50hz.

Everything you need from a PAL ROM can be patched in already. Please submit missing PAL features for into the Rev0 US ROM instead.

If it's not OoT. We need patches. There are simply no patches being made for PAL, so PAL ROMs are not supported due to lack of content.

The Minish Cap contains both PAL and US and either ROM can be provided. In fact you can provide PAL ROMs for most games which gets automaticially converted to US.

Please clarify which games need PAL support, because the assumption here is OoT/MM.

**Reply by AMiR-DU-Bled** [reply]
ok I didn't know I can change the language so I will try it and see if it works

**Reply by AMiR-DU-Bled** [reply]
like mario kart 64, paper mario and mario 64 in french beacause I cant find the option to

**Reply by Admentus64** [reply]
Those games lack support for PAL content yes.

Maybe in the far future. Dunno. No plans cus there aren't any Patreon requests or commissions for this.

**Reply by AMiR-DU-Bled** [reply]
oh ok I understand and you rely on donnations to maintain the project but its incredible ! keep the work man.
